components/theme/ThemedIconButton.tsx

```diff
@@ -10,6 +10,7 @@ interface ThemedIconButtonProps {
   style?: ViewStyle;
   size?: number;
   color?: string;
+  testID?: string;
```

Comment:
Actually you don't need this as one the last PR to master's branch implements this by extending ThemedIconButtonProps with IconProps.